### PR TITLE
New text elements for WebFlow.

### PR DESCRIPTION
**Pull request by zcgandcomp**

### CODE REVIEW

File: powerauth-webflow/package.json [hunk 1]
```diff
@@ -19,43 +19,44 @@
...
+    "axios": "^0.27.2",
+    "bluebird": "^3.7.2",
+    "intl": "^1.2.5",
+    "lodash": "^4.17.21",
     "react": "^16.14.0",
     "react-bootstrap": "^0.33.1",
-    "react-select": "1.3.0",
+    "react-device-detect": "^2.2.2",
     "react-dom": "^16.14.0",
+    "react-flags-select": "^2.2.3",
```
Comment (romanstrobl) [inline powerauth-webflow/package.json:31]: Let's revert all changes except for `react-flags-select`, because many of the dependencies are a downgrade.

**Review by romanstrobl — CHANGES_REQUESTED** [review]
Looks OK, let's just revert the dependency changes, we just need to add 1 new dependency and leave the rest as they are.

**Review by romanstrobl — APPROVED** [review]
Looks OK.

File: powerauth-webflow/package.json (hunk 1 above)
Comment (zcgandcomp) [reply powerauth-webflow/package.json]: package.json updated and currently contains no downgrades.

**Review by romanstrobl — APPROVED** [review]

**Pull request merged**